### PR TITLE
docs: import pgBackRest docs

### PR DESCRIPTION
**Pull request by gsong**
*Note: This PR is a bit confusing - it began as #926, and we inadvertently merged it without doing a review pass on the structure & metadata. So it got reverted in #928 & pulled into this branch.*

The changes beyond what was in #926 consist of:

- Adding `navTitle`, `description`, `product` and `tags` metadata where appropriate
- Merging index.mdx and 01_Introduction.mdx 
- minor correction to links (both case studies linked to #restore with the text and context "backup" - I'm assuming this was unintended?)

Most of these can be found in 2223729

### CODE REVIEW

**Comment by josh-heyer** [comment]
Sorry for the bumpy road here, @pgstef - if you wouldn't mind scanning the changes noted above in the test environment (https://edb-docs-adv-content-pg-alxu6q.herokuapp.com/supported-open-source/pgbackrest/), this can be merged in now if there are no objections. In the future, we should be a bit more on the ball with this stuff - when adding new content we just need to ensure that the correct metadata is in place for search & navigation to work optimally.

FWIW: this motivated me to make another change I'd been putting off - making the auto-generated "card deck" index somewhat configurable (#935).

File: advocacy_docs/supported-open-source/pgbackrest/05-retention_policy.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: 'Retention Policy'
+description: "Configuration options and considerations for the removal of obsolete backups"
```
Comment (pgstef) [inline advocacy_docs/supported-open-source/pgbackrest/05-retention_policy.mdx:3]: -> obsolete backups and WAL archives

File: advocacy_docs/supported-open-source/pgbackrest/06-use_case_1.mdx [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+Check the configuration and the archiving process:
+
+```bash
+$ pgbackrest --stanza=demo check
+```
+
+### Backup and Restore
+
+See the **Quick start** [backups](03-quick_start#backups) sections for more details about those two commands.
```
Comment (pgstef) [inline advocacy_docs/supported-open-source/pgbackrest/06-use_case_1.mdx:174]: It seems it didn't pulled the latest sources with both links to backups and restore...

```
See the **Quick start** [backups](03-quick_start#backups) and [restore](03-quick_start#restore) sections for more details about those two commands.
```

File: advocacy_docs/supported-open-source/pgbackrest/07-use_case_2.mdx [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+Check the configuration and the archiving process:
+
+```bash
+$ sudo -u pgbackrest pgbackrest --stanza=demo check
+```
+
+### Backup and Restore
+
+See the **Quick Start** [Backups](03-quick_start#backups) pages for more details about those two commands.
```
Comment (pgstef) [inline advocacy_docs/supported-open-source/pgbackrest/07-use_case_2.mdx:162]:
```
See the **Quick start** [backups](03-quick_start#backups) and [restore](03-quick_start#restore) sections for more details about those two commands.
```

**Review by pgstef — CHANGES_REQUESTED** [review]
Hi,

I didn't read the entire documentation, assuming you didn't change its content.
However, I made a few minor formatting comments. I apparently can't push it myself to this branch. Please, fix those before merging.

Also, in the provided url, the links seems broken. Example:
- https://edb-docs-adv-content-pg-alxu6q.herokuapp.com/docs/supported-open-source/pgbackrest/02-installation/ -> extra "docs".

I guess it's due to the poc deployment?

File: advocacy_docs/supported-open-source/pgbackrest/06-use_case_1.mdx (hunk 2 above)
Comment (josh-heyer) [reply advocacy_docs/supported-open-source/pgbackrest/06-use_case_1.mdx]: That's... peculiar

**Comment by josh-heyer** [comment]
Ok, I've made the corrections you outlined. Link breakage should be fixed (bug in our build process) - will verify in next deploy and then merge.

**Review by josh-heyer — APPROVED** [review]
Think this is good to meet develop; further work can continue separately

**Pull request merged**